src/content/resources/java/class.md

```diff
@@ -0,0 +1,141 @@
+---
+authors:
+  - "AstronautEVA#0331"
+title: "What is a class?"
+created_at: 2019/10/20
+
+---
+# What is a class?
+
+Oftentimes in programs, it is necessary to represent objects from the real world as computer data. 
+For example, if someone is writing a program that simulates a zoo, they will need to create some animals.
+A class is a way to write the blueprint of whatever you are trying to represent; in our case it would be a blueprint of an animal.
+
+Let us investigate what an animal class might look like.
+First we will declare attributes that an animal would have. There are many attributes we could list, but let's choose 3.
+The animal will have a 
+* name (because zoos name their animals)
+* age
+* weight
+
+All of these words are nouns which is a hint that they should be implemented as variables.
+```java
+public class Animal {
+  private String name;
+  private int age;
+  private float weight;
+}
+```
+
+Notice that we have only declared these variables, we did **not** initialize them (set them equal to a value). 
+Remember this is only the blueprint (the recipe) of an animal, not a specific animal. Later we will create specific animals from this blueprint.
+
+Okay so we have added attributes to our animal which is great, but we want our animal to actually be able to *do* things. Let's pick 2
+things we want our animal to do.
+Our animal should
+* move
+* sleep
+
+These are verbs which means they will be implemented as methods.
```

Comment:
Even though it is often the case, something being named a verb does not necessarily mean they will be implemented as methods.